cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -67,7 +67,7 @@ func GetStaticPodSpecs(cfg *kubeadmapi.InitConfiguration, k8sVersion *version.Ve
 			ImagePullPolicy: v1.PullIfNotPresent,
 			Command:         getControllerManagerCommand(cfg, k8sVersion),
 			VolumeMounts:    staticpodutil.VolumeMountMapToSlice(mounts.GetVolumeMounts(kubeadmconstants.KubeControllerManager)),
-			LivenessProbe:   staticpodutil.ComponentProbe(cfg, kubeadmconstants.KubeControllerManager, 10252, "/healthz", v1.URISchemeHTTP),
+			LivenessProbe:   staticpodutil.ComponentProbe(cfg, kubeadmconstants.KubeControllerManager, 10257, "/healthz", v1.URISchemeHTTPS),
```

Comment:
Please fix also the test cases for self-hosting pod mutation for using new port numbers (see `cmd/kubeadm/app/phases/selfhosting/selfhosting_test.go`)

Reply:
@fabianofranz, yes i can.

It looks as if the selfhosting_test hasn't seen much work recently as it's still running `1.7.4`. So changes to the test cases will involve more work. To verify, are changes that large appropriate for this PR?

Reply:
I think that changing only the `bind-address` bit is ok
All the rest should go into another PR (or at least an issue should be opened to track this)